src/strucclustutils/structcreatedb.cpp

```diff
@@ -74,6 +74,17 @@ static inline bool compareByFirst(const std::pair<T, U>& a, const std::pair<T, U
     return a.first < b.first;
 }
 
+std::string removeMODEL(const std::string& input) {
+    size_t modelIndex = input.find("MODEL");
+    if (modelIndex == std::string::npos)
```

Comment:
Please don't omit curly brackets `{}` for single line `if` statements.

Reply:
Done! thanks